src/ucp/wireup/wireup_cm.c

```diff
@@ -482,6 +482,9 @@ void ucp_cm_client_restore_ep(ucp_wireup_ep_t *wireup_cm_ep, ucp_ep_h ucp_ep)
         }
     }
 
+    /* TMP EP is not an owner of local EP ID */
+    ucs_assert(ucp_ep_local_id(tmp_ep) == ucp_ep_local_id(ucp_ep));
+    ucp_ep_ext_control(tmp_ep)->local_ep_id = UCP_EP_ID_INVALID;
```

Comment:
is it really needed, given that we destroy `tmp_ep` on the line below?

Reply:
I just wanted to make sure that `ucp_ep_destroy` function won't try to destroy `local_ep_id` for TMP EP (it will fail an assertion in `ucp_ep_release_id()` which expects that ID is not INVALID one), because a normal UCP EP (i.e. owner of this TMP EP) is responsible for it.
this is just for debugging purposes and was helpful for me to debug problem, when TMP EP released an EP ID, but UCP EP can't do it, because its EP ID is unknown for PTR MAP.

Reply:
ok